docs/testing-operators.md

```diff
@@ -1,120 +1,404 @@
+
+### 1. Create the OperatorSource
+
+An `OperatorSource` object is used to define the external datastore we are using to store operator bundles. More information including example can be found in the documentation included in the `operator-marketplace` [repository](https://github.com/operator-framework/operator-marketplace#operatorsource).
+
+**Replace** `johndoe` in `metadata.name` and `spec.registryNamespace` with your quay.io username in the example below and save it to a file called `operator-source.yaml`.
+
+```
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: johndoe-operators
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: johndoe
+  displayName: "John Doe's Operators"
+  publisher: "John Doe"
+```
+
+Create the object:
+
+```
+kubectl apply -f operator-source.yaml
+```
+
+Check if the `OperatorSource` was processed correctly:
+
+```
+kubectl get operatorsource johndoe-operators -n openshift-marketplace
+
+NAME                TYPE          ENDPOINT              REGISTRY   DISPLAYNAME            PUBLISHER   STATUS      MESSAGE                                       AGE
+johndoe-operators   appregistry   https://quay.io/cnr   johndoe    John Doe's Operators   John Doe    Succeeded   The object has been successfully reconciled   30s
+```
+
+### 2. Find your Operator in the OperatorHub UI
+
+Go to your OpenShift UI and find your Operator by filtering for the *Custom* category:
+
+![Find your Operator in OperatorHub](images/my-operator-in-hub.png)
+
+### 3. Install your Operator from OperatorHub
+
+To install your Operator simply click it's icon and in the proceeding dialog click *Install*.
+
+![Install your Operator from OperatorHub](images/my-operator-install.png)
+
+ You will be asked where to install your Operator. Select either of the desired installation modes, if your Operator supports it and then click *Subscribe*
+
+![Install your Operator from OperatorHub](images/my-operator-subscription.png)
+
+You will be forwarded to the *Subscription Management* section of the OLM UI and after a couple of moments your Operator will be transitioning to *Installed*.
+
+![Subscribe to your Operator from OperatorHub](images/my-operator-subscribed.png)
+
+### 4. Verify Operator health
+
+Change to the *Installed Operators* section in the left-hand navigation menu to verify your Operator's installation status:
+
+![See your installed Operator](images/my-operator-installed.png)
+
+It should have transitioned into the state *InstallationSucceeded*. You can now test it by starting to use it's APIs.
+
+## Testing with scorecard
+
+If your Operator is up and running you can verify it's working as intended using it's APIs. Additionally you can run [operator-sdk](https://github.com/operator-framework/operator-sdk/blob/master/doc/test-framework/scorecard.md)'s `scorecard` utility for validating against good practice and correctness of your Operator.
```

Comment:
nit: `its` is possessive, `it's` is a contraction for `it is`. In this sentence, you are using `it's` to mean both.

Should be:

`If your Operator is up and running you can verify it's working as intended using its APIs`